### DISCUSSION TITLE
Soundtrack

### DISCUSSION THREAD

**Discussion by TheGameSquad22** [terrarium-earth/Ad-Astra#417] [Q&A] [answered]
I want to listen to the space station soundtrack in my off time, it sounds so serene and encompassing, but i cant find it anywhere on youtube or other music streaming platforms that it could be on. Can you give me a link to the soundtrack file or youtube/spotify post for the official soundtrack?

**Comment by AlexNijjar** [comment]
what space station soundtrack?

**Reply by TheGameSquad22** [reply]
It sounds like the normal minecraft overworld soundtrack, but slowed down and with a bit of reverb

**Reply by TheGameSquad22** [reply]
im finding it to play in the earth_orbit dimension

**Reply by AlexNijjar** [reply, marked as answer]
Oh. all it is is the minecraft music slowed down a bit. I can't redistribute it but you can create it yourself by putting the sounds in any audio editor and slowing it down. Here's a download link https://mcasset.cloud/1.20.4/assets/minecraft/sounds/music/game

**Reply by TheGameSquad22** [reply]
it sounds similar to the leitmotif references in the End dimension.
